tools/pot/docs/AccuracyAwareQuantizationUsage.md

```diff
@@ -11,27 +11,29 @@
+> **NOTE**: The Accuracy-aware Quantization algorithm behavior is different for GNA `target_device`. It is searching for the best configuration, selecting between INT8 and INT16 precisions for weights of each layer. The algorithm works for the `performance` preset only. This algorithm is not helpful, for the `accuracy` preset, since the whole model is already in INT16 precision.
 
-A script for Accuracy-aware Quantization includes four steps:
-1. Prepare data and dataset interface
-2. Define accuracy metric
-3. Select quantization parameters
-4. Define and run quantization process
+Script for Accuracy-aware Quantization includes four steps:
+1. Prepare data and dataset interface.
+2. Define accuracy metric.
+3. Select quantization parameters.
+4. Define and run quantization process.
 
 ## Prepare data and dataset interface
-This step is the same as in the case of [Default Quantization](@ref pot_default_quantization_usage). The only difference is that `__getitem__()` method should return `(data, annotation)` or `(data, annotation, metadata)` where `annotation` is required and its format should correspond to the expectations of the `Metric` class. `metadata` is an optional field that can be used to store additional information required for post-processing.
+This step is the same as in the case of [Default Quantization](@ref pot_default_quantization_usage). The only difference is that `__getitem__()` method should return `(data, annotation)` or `(data, annotation, metadata)`. In there, `annotation` is required and its format should correspond to the expectations of the `Metric` class. The `metadata` is an optional field that can be used to store additional information required for post-processing.
 
 ## Define accuracy metric
-To control accuracy during the optimization a `openvino.tools.pot.Metric` interface should be implemented. Each implementation should override the following properties:
+To control accuracy during the optimization a `openvino.tools.pot.Metric` interface should be implemented. Each implementation should override the following properties and methods:
+
+**Properties**
 - `value` - returns the accuracy metric value for the last model output in a format of `Dict[str, numpy.array]`.
 - `avg_value` - returns the average accuracy metric over collected model results in a format of `Dict[str, numpy.array]`.
-- `higher_better` should return `True` if a higher value of the metric corresponds to better performance, otherwise, returns `False`. Default implementation returns `True`.
+- `higher_better` if a higher value of the metric corresponds to better performance return `True` , otherwise, `False`. Default implementation returns `True`.
```

Comment:
```suggestion
- `higher_better` if a higher value of the metric corresponds to better performance, return `True` , otherwise, `False`. Default implementation returns `True`.
```